src/components/helpers/RwdHelper.jsx

```diff
@@ -32,13 +32,17 @@ const RwdHelper = ({hide, children}: PropsType) => {
 
   const hideClass = `sg-hide-for-${hide}`;
 
-  if (typeof children === 'string') {
+  if (
+    typeof children === 'string' ||
```

Comment:
@matzimowski I think you could type it with flow https://flow.org/en/docs/react/children/